modules/ROOT/pages/config-alerts.adoc

```diff
@@ -1,41 +1,92 @@
-= Set up Alerts for CPU and Memory Usage
+
+= Configure Alerts for Anypoint Private Cloud
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) sends alert emails when CPU and memory usage reach a certain threshold:
+Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) provides built-in alerts that are triggered when a condition specified in any alert definition is detected.
+
+Measurements are stored in https://www.influxdata.com/products/influxdb-overview[Influxdb] and read by https://www.influxdata.com/time-series-platform/kapacitor[Kapacitor]. Kapacitor sends emails when an alert is triggered.
+
+== Alert Definitions
+
+[%header%autowidth.spread]
+|===
+| Name | Description | Threshold | Configmap
```

Comment:
Is `configmap` syntax? You would not format it in the column heading, of course, unless you call it Config Map or Configuration Map. I wonder if it should be lowercase, however, if it really is syntax.